test/common/upstream/health_checker_impl_test.cc

```diff
@@ -220,6 +220,29 @@ class HttpHealthCheckerImplTest : public testing::Test {
     });
   }
 
+  void setupServiceValidationWithCustomHostValueHC(const std::string& host) {
+    std::string json = fmt::format(R"EOF(
```

Comment:
If we drop v1 support, let's make this a v2 YAML config for testing?

Reply:
OK, I have it in v2 YAML on b3f2c8f. Thanks for reviewing @mattklein123!